user_funcs.py

```diff
@@ -116,3 +117,246 @@ def login(conn) -> tuple[str, int]:
+        user_ID = str(user_ID)
+        #Loop that prompts user for information
+        print("\ncreate_collection function running...")
+        decision = int(input_utils.get_input_matching("\n1 - Create an empty collection \n2 - Create a new collection that will be filled with movies: ", regex='[12]'))
+        #Creates an empty collection
+        if decision == 1:
+            collection_Name = input("What would you like to name your new collection?\n")
+            curs.execute("INSERT INTO moviecollection (name, madeby) VALUES (%s, %s)", (collection_Name, user_ID))
+            
+            curs.execute("SELECT name FROM moviecollection WHERE madeby = %s", (user_ID,))
+            collection_Name = curs.fetchall()
+
+        elif decision == 2:
+            #Insert new collection with new name
+            collection_Name = input("What would you like to name your new collection?\n")
+            curs.execute("INSERT INTO moviecollection (name, madeby) VALUES (%s, %s)", (collection_Name, user_ID))
+            #gets the id of the newly add collection
+            curs.execute(collection_ID_query, ())
+            collection_ID_results = int(curs.fetchone()[0])
+            collection_ID = str(collection_ID_results)
+            movie_id = 0
+            #adding movies to the collection
+            while loop_state:
+                #adding movies that are in the database to a collection when making a new collection
+                break_loop = 0
+                print("what movie would you like to add?")
+                movie_id = movie_funcs.browse_movies(conn)
+                if movie_id == -1:
+                    print("Was not able to get movieID to add to collection.\nWill Exit...")
+                    curs.execute("DELETE FROM moviecollection WHERE name = %s AND madeby = %s", (collection_Name, user_ID,))
+                    return
+                else:
+                    movie_ID = str(movie_id)
+                    curs.execute(add_movie_collection_query, (movie_ID, collection_ID))
+                break_loop = int(input_utils.get_input_matching("\n0 - Continue adding movies\n1 - Stop adding movies: "))
+                if break_loop == 0:
+                    pass
+                elif break_loop == 1:
+                    break
+                else:
+                    while True:
+                        print(f"Invalid Input of {break_loop}, please enter a valid input")
+                        break_loop = int(input_utils.get_input_matching("""\n0 - Continue adding movies\n1 - Stop adding movies: """))
+                        if break_loop == "0" or "1":
+                            break
+                if break_loop == 0:
+                    pass
+                else:
+                    break
+            else:
+                print("Not an acceptable answer for decision! Ending function")
+                return
+    conn.commit()
+    print(f"Finished create_collection function!!")
+
+
```

Comment:
NIT:
```suggestion
```
I think the python style guide recommends two lines between function, but this is the most asanine of nitpicks so don't take it too seriously :P